caffe/CMakeLists.txt

```diff
@@ -51,6 +51,8 @@ include(cmake/Dependencies.cmake)
 
 find_package(OpenBabel2 REQUIRED)
 
+find_package(OpenMP)
```

Comment:
Can this be made a soft requirement? (Not an error if it isn't found, but gninadream doesn't build - like with rdkit and gninavis).